.github/workflows/robustness-template-arm64.yaml

```diff
@@ -0,0 +1,72 @@
+name: Reusable Robustness Workflow
+on:
+  workflow_call:
+    inputs:
+      etcdBranch:
+        required: true
+        type: string
+      count:
+        required: true
+        type: number
+      testTimeout:
+        required: false
+        type: string
+        default: '30m'
+      artifactName:
+        required: true
+        type: string
+      runs-on:
+        required: false
+        type: string
+        default: "['ubuntu-latest']"
+permissions: read-all
+
+jobs:
+  goversion:
+    uses: ./.github/workflows/go-version.yaml
+  test:
+    timeout-minutes: 210
+    runs-on: ${{ fromJson(inputs.runs-on) }}
+    needs: goversion
+    container: golang:1.19-bullseye
+    defaults:
+      run:
+        shell: bash
```

Comment:
You do not configure the volumes, so I think the artifact may not be correctly uploaded when robustness runs into any issues?

Reply:
Based on [running-jobs-in-a-container#overview](https://docs.github.com/en/actions/using-jobs/running-jobs-in-a-container#overview), the test script and `actions/upload-artifact` will run in separate containers, and accordingly they will have different local filesystems & volumes if you do not mounts the host paths into the containers. 

I think you can verify this by intentionally fail the test.

```
If you have steps that use both script and container actions, the container actions 
will run as sibling containers on the same network with the same volume mounts.
```

Reply:
Have intentionally failed test, can you please check https://github.com/etcd-io/etcd/actions/runs/5066334564/jobs/9096012829?pr=15927 ?

Reply:
Confirmed that the artifact was successfully uploaded. It seems that I misunderstood the meaning of `container actions`, and the `actions/upload-artifact` isn't a container action, so all steps are running in the same container.

Thanks for the confirmation.

Reply:
Was a good question and worth double checking, thanks! 🙏🏻